README.md

```diff
@@ -4,7 +4,9 @@ This repository contains links to Camunda Platform 8 resources, the offical rele
 
 :warning: **Docker Compose is only recommended for local development.** :warning:
 
-For more information about Self-Managed, including additional development installation options, see our [documentation](https://docs.camunda.io/docs/next/self-managed/about-self-managed/).
+We recommend using [SaaS](https://camunda.com/get-started/) or [Helm/Kubernetes](https://docs.camunda.io/docs/next/self-managed/platform-deployment/helm-kubernetes/overview/) for development.
+
+For more information about Self-Managed, including additional [development installation options], see our [documentation](https://docs.camunda.io/docs/next/self-managed/about-self-managed/).
```

Comment:
Did you mean to directly link to development installation options here?

Reply:
Whoops, good catch! Yes!